### ISSUE TITLE
Browserify unnecessary shim?

### ISSUE THREAD

**Issue by motiazu** [IjzerenHein/famous-flex#71]
Hi,
I started working with Browserify in order to load my modules, and the famous-flex package keeps failing.

The line that fails is in LayoutController (because this is the first component my code requests via require(...)).
The following line is generated:
```
var Utility = typeof window !== 'undefined' ? window.famous.utilities.Utility : typeof global !== 'undefined' ? global.famous.utilities.Utility : null;
```
window.famous is undefined, thus an error occurs.

After going nuts over this weird thing, I eventually removed the shim process from the transform in famous-flex package.json.
I re-browserified and the line became:
```
var Utility = require('famous/utilities/Utility');
```

After that, everything worked great. I'd hate to leave modified code of a dependency in my app, and there's a good chance I misconfigured browserify somehow...
This is my package.json:
```
{
  "browserify": {
    "transform": [
      "deamdify"
    ]
  },
  "dependencies": {
    "famous": "^0.3.5",
    "famous-flex": "0.1.9"
  }
}
```
Any idea why the shim ruins the build?

**Comment by IjzerenHein** [comment]
Are you using the latest version from master?

**Comment by motiazu** [comment]
I'm using "0.1.9", Looks like the current version to me here.
https://github.com/IjzerenHein/famous-flex/blob/master/package.json

**Comment by IjzerenHein** [comment]
Yes, I was just wondering whether you are using the latest version from bower/npm or from the master branch.

**Comment by motiazu** [comment]
Oh, well npm it is. Aren't these in sync?

**Comment by IjzerenHein** [comment]
No, the master is ahead of the npm/bower version. Various commits have been made to the master concerning the global-famous-flex.js file. I want people to field test these first prior to releasing them as a package on npm/bower.
Could you use the current dist/global-famous-flex.js file from the master and see whether that one works for you?

**Comment by motiazu** [comment]
Tried the master, still get the same problem. Also solved by removing the shim transformation.

**Comment by IjzerenHein** [comment]
And what about this version released by Sidney in this pull request?
https://github.com/IjzerenHein/famous-flex/pull/70

**Comment by motiazu** [comment]
Same bug, same solution. I'm suspecting maybe when the shim was added it was needed, but famo.us had changed since then and maybe it's not needed anymore.

**Comment by IjzerenHein** [comment]
Could it be because they removed the 'src' folder in 0.3.5?

**Comment by motiazu** [comment]
Very possible, this also made famousify redundant.

**Comment by IjzerenHein** [comment]
Hi, I just released a new version and updated the NPM package v0.2.0.
Do you still have issues with the new version?

**Comment by motiazu** [comment]
Seems to work yo :+1:

**Comment by IjzerenHein** [comment]
Excellent!! :+1:

**Comment by motiazu** [comment]
Whoops... still happens. I guess my original check wasn't good, but now only removing the shim resolves the issue like before.

**Comment by IjzerenHein** [comment]
@motiazu Do you mean just removing the shim from the package.json?
Could you post the changes that you needed to make to get it working?

**Comment by motiazu** [comment]
Before:
`
"browserify": {
    "transform": [
      "browserify-shim",
      "deamdify"
    ]
  },
`
After:
`
"browserify": {
    "transform": [
      "deamdify"
    ]
  },
`

**Comment by IjzerenHein** [comment]
Hi, I've just released a new version of famous-flex (v0.3.2). It contains all new generated `/dist` files to meet different kind of integration scenarios.
I've also created a test project, which showcases how to integrate famous-flex using different technologies (requirejs, global, wepback, browserify:
https://github.com/IjzerenHein/famous-lib-tester
